std/typecons.d

```diff
@@ -6424,7 +6424,7 @@ if (!is(T == class) && !(is(T == interface)))
             import std.algorithm.mutation : moveEmplace;
 
             allocateStore();
-            moveEmplace(source, _store._payload);
+            () @trusted { moveEmplace(source, _store._payload); }();
```

Comment:
what about `inisialize` just above this? Is that covered by tests?

Reply:
I added a test to ensure `initialize` is `@safe`.